string robot_id
geometry_msgs/Point point
float64 distance_neg
